clang/test/CodeGen/inline-asm-fp-to-int.c

```diff
@@ -0,0 +1,8 @@
+// RUN: %clang_cc1 %s -emit-llvm -o /dev/null
+
+unsigned test(float f)
```

Comment:
What IR does this generate?

Reply:
This test is using -emit-llvm, but the patch is in SelectionDAG. SelectionDAG doesn't run with -emit-llvm

Reply:
Thanks for the review. You are right, indeed the test was pointless, the problem is with CodeGen. Updated the issue accordingly.